### ISSUE TITLE
[ACCESSIBILITY] Color contrast insufficient for Title and back arrow of "Self Skills Evaluation"

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#4020]
**Describe the bug**
In the “Skills Self Evaluation”, “<-” the back arrow and title of screen “Skills Self Evaluation” do not meet color contrast ratio. Text that is too faint can cause problems for users who are color blind or have low vision. WCAG requires a minimum color contrast ratio of at least 4.5:1 for normal text, and 3:1 for large text (18pt and above, or 14pt bold).
[WCAG 1.4.3 Contrast (Minimum)](https://www.w3.org/TR/WCAG21/#contrast-minimum)

**To Reproduce
Steps to reproduce the behavior:**
1.	Open app, logged in as student
2.	Navigate to Skills Self Evaluation

**Expected behavior**
The title of screen would be visible with contrast of at least 4.5:1 for normal text, and 3:1 for large text (18pt and above, or 14pt bold).

**Actual behavior**
Foreground text:1B1F1F
Background text: 0C2042

**Screenshots**
![Screenshot_20240320-163528](https://github.com/rokwire/illinois-app/assets/82892873/42042157-06bf-40d8-bf3f-2d4bbc561b85)

**Testing Environment & Devices (please complete the following information):**
•	OS: Android Version 13
•	Device: Google Pixel 4a

**Comment by TodorBachvarov** [comment]
Fixed in v5.2.3

**Comment by acovher2** [comment]
Verified as fixed in 6.0.6 apk
